packages/block-editor/src/components/block-list/style.scss

```diff
@@ -278,12 +282,15 @@
 
 	// Wide and full-wide.
 	&[data-align="full"],
-	&[data-align="wide"] {
+	&[data-align="wide"],
+	&.alignfull,
+	&.alignwide {
```

Comment:
Hm, ideally this should be provided by the theme, no? I mean, ideally we shouldn't do any special handling for these classes.